### DISCUSSION TITLE
Does the tsconfig.json in the docs need to be updated?

### DISCUSSION THREAD

**Discussion by txtyash** [vitest-dev/vitest#4064] [Q&A]
tsconfig.json example in the docs is:
```ts
// tsconfig.json
{
  "compilerOptions": {
    "types": [
+     "vitest/importMeta"
    ]
  }
}
```
Shouldn't that be 
```ts
// tsconfig.json
{
  "compilerOptions": {
    "module": "es2022",
    "moduleResolution": "Node",
    "types": [
      "vitest/importMeta"
    ]
  }
}
```
in reference to this issue: https://stackoverflow.com/questions/70440505/ts1343-the-import-meta-meta-property-is-only-allowed-when-the-module-opti

**Comment by sheremet-va** [comment]
There is no need to update tsconfig to make this work in Vitest. Only if you have TS errors.
